lib/AST/ASTImporter.cpp

```diff
@@ -5660,16 +5660,37 @@ ASTNodeImporter::VisitFunctionTemplateDecl(FunctionTemplateDecl *D) {
         // name conflict), because FunctionTemplates can possibly overload.
       } // template
     }   // for
-  }
 
-  auto ParamsOrErr = import(D->getTemplateParameters());
-  if (!ParamsOrErr)
-    return ParamsOrErr.takeError();
+#if 0
```

Comment:
This is another type of solution but I do not like it, the current way seems more simple and robust. In the final PR this code should be removed.